src/components/LandUseVocabulary.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import _ from 'lodash'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import Tooltip from 'react-bootstrap/Tooltip'
+import Button from 'react-bootstrap/Button'
+
+import terms from '../assets/keywords'
+
+import './LandUseVocabulary.css';
+
+class LandUseVocabulary extends Component {
+  
+  state = {
+    usesString: this.props.string,
+  };
+
+  render() {
+    const { usesString } = this.state;
+    // splitting the uses
+    let uses = _.split(usesString, ',')
```

Comment:
We don't need the lodash dependency here. You may write it
```suggestion
    return usesString.split(',').map(use=>this.renderUse(use))
```

Reply:
done